source/user/eph.rst

```diff
@@ -1,8 +1,8 @@
 .. _user_eph:
 
-**********************
-Electron Phonon Matrix
```

Comment:
Are you redoing a change in master here or have you changed this as part of this PR?

Reply:
This was another intentional small change from Tim